src/Infrastructure/TemplateBasedMailer.php

```diff
@@ -36,4 +36,17 @@ public function sendMail( EmailAddress $recipient, array $templateArguments = []
 		);
 	}
 
+	/**
+	 * @throws \RuntimeException
+	 */
+	public function sendMailToOperator( EmailAddress $replyToAddress, array $templateArguments = [] ) {
```

Comment:
I was not able to do this now yet... can do it tomorrow. If you agree with what I propose, feel free to merge already, then we can do this change in a follow up.